hack/ostree_tag.sh

```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
 if pkg-config ostree-1 2> /dev/null ; then
-	echo ostree
```

Comment:
With this, `containers/storage/pkg/ostree/` will now be disabled, affecting the `vfs` and `overlay` drivers.
- Is that intentional? I can’t see any explicit reference to this in the discussion.
- Independently, is it desirable? (@nalind ?)

Reply:
The `github.com/containers/storage` `ostree` package also links with the native libraries, so if they're not present, and we don't want the build to fail because of that, the build tag would need to be removed.

For the case where detecting that the libraries are present means we enable the feature in the image library, I'd expect us to want to define the `ostree` build tag as well.